client/src/views/Dashboard/Dashboard.js

```diff
@@ -62,9 +62,28 @@ const prepareHistogramTransactionData = histogram => Object.keys(histogram).map(
 const isMobileOnly = window.matchMedia('(max-width: 480px)').matches
 
 // chart configs for mobile devices
-const lineChartTickRotation = isMobileOnly ? -45 : 0
-const mobilePieChartProps = isMobileOnly ? { width: 400, height: 250 } : {}
 const lineChartDataLimiter = isMobileOnly ? 10 : 20
+const lineChartTickRotation = isMobileOnly ? -90 : 0
+const mobilePieChartProps = isMobileOnly ? {
```

Comment:
```
!isMobile ? {} : {
/// object hrre
}
```

Reply:
could we make point circles a bit lesser on a line chat ?
and move ' count ' legend to the top side (it takes some width) ?

Reply:
or just remove as screen title is 'Daily **count** of ...'

Reply:
Done